pkg/operator/csidriveroperator/driver_starter.go

```diff
@@ -196,8 +196,13 @@ func (dsrc *driverStarterCommon) sync(ctx context.Context, syncCtx factory.SyncC
 			return err
 		}
 
+		isInstalled, err := dsrc.commonClients.OperatorClient.IsOperatorInstalled(ctrl.operatorConfig.CSIDriverName)
+		if err != nil {
+			return err
+		}
+
```

Comment:
Please move it to `!ctrl.running` branch below, so it's called only when needed.

Reply:
Done.